boefjes/boefjes/plugins/kat_dns/normalize.py

```diff
@@ -152,6 +155,14 @@ def register_record(record: DNSRecord) -> DNSRecord:
                         )
                     )
 
+                if isinstance(rr, CAA):
+                    recordvalue = str(rr)
```

Comment:
`record_value`, `value` (or `record`)

Reply:
I'm not hinting about the content of the variable, but simply the name based on PEP naming conventions. "Record value" is a two-word phrase and should therefore be `record_value`